client/store/product/productList.js

```diff
@@ -11,26 +12,14 @@ export const setProductList = (productList) => ({
 // fetchProductList is a thunk that grabs products from our backend paginated
 // route /api/products and attaches relevant queries from productPagination and
 // productFilters stored in state
-export const fetchProductList = () => {
+export const fetchProductList = (query) => {
   return async (dispatch, getState) => {
-    const { productPagination, productFilters } = getState();
-    let query = [];
-    for (const paramKey in productPagination) {
-      if (productPagination[paramKey] !== '' && paramKey !== 'maxPage') {
-        query.push(`${paramKey}=${productPagination[paramKey]}`); //ex. page=1
-      }
-    }
-    for (const paramKey in productFilters) {
-      if (productFilters[paramKey] !== '') {
-        query.push(`${paramKey}=${productFilters[paramKey]}`);
-      }
-    }
-    query = query.join('&');
-    console.log(query);
```

Comment:
Since we're now using the query on the component side, too, I turned this into a utility function and we now just pass the query into the `fetchProductList` thunk.